src/Jmikola/WildcardEventDispatcher/WildcardEventDispatcher.php

```diff
@@ -33,7 +33,11 @@ public function dispatch($eventName, Event $event = null)
     {
         $this->bindPatterns($eventName);
 
-        return $this->dispatcher->dispatch($eventName, $event);
+        if ($event === null) {
+            $event = new Event();
+        }
+
+        return $this->dispatcher->dispatch($event, $eventName);
```

Comment:
If Symfony 4.3+ changed the order of the `dispatch()` parameters for 5.0, should `WildcardEventDispatcher::dispatch()` also be updated to relax its signature and support both argument orders as is done in Symfony's [`EventDispatcher::dispatch()`](https://github.com/symfony/symfony/blob/dfa118f2536958017d89a28b2b0ac3234ee0e3f9/src/Symfony/Component/EventDispatcher/EventDispatcher.php#L51) implementation?

As-is, this library is requiring Event to always be passed as the second argument per its type hint.

Reply:
I suggest not to support backward compatibility. Let's create a version `2.0.0` that will be compatible with Symphony `^4.3 | ^5.0`.